pkg/loop/internal/core/services/reportingplugin/ocr3/reporting_plugin_service.go

```diff
@@ -96,9 +96,6 @@ func (o *ReportingPluginServiceClient) NewReportingPluginFactory(
 		deps.Add(keyValueStoreRes)
 
 		relayerSetServer, relayerSetServerRes := relayerset.NewRelayerSetServer(o.Logger, relayerSet, o.BrokerExt)
-		if err != nil {
```

Comment:
This was comparing an error that's already previously checked, `NewRelayerSetServer` doesn't return an err